src/Model/RecordLocale.php

```diff
@@ -293,6 +293,35 @@ public function IsPublished()
         return true;
     }
 
+    /**
+     * Check for published and localised record
+     * useful when determining what data is available in which locale
+     *
+     * @return bool
+     */
+    public function IsPublished()
```

Comment:
Is this method just the same but with frontend_publish_required ignored?

Reply:
Suggestion: instead of a new method, simplify (and with clear documentation) add a `$inLocale = false` property.
```php
/**
 @param bool $inLocale Set to true if this record must be published in the specified locale. If frontend_publish_required is false (and $inLocale is false) then this page is considered live.
*/
```

Reply:
I wouldn't be against having a new method that simply called the other with a different flag, but not one with 90% of the same code please.